src/client.js

```diff
@@ -118,6 +119,15 @@ class Client {
   }
 
   /**
+   * A wrapper around the constructor, for backwards compatability.
```

Comment:
this isn't for backward compatibility. This is the primary entry point for most consumers.